docs/tasks_volumeprovisioning.md

```diff
@@ -378,7 +378,48 @@ kubectl delete pod percona-b98f87dbd-nqssn
 
 13. Application pod must be in running state and you can use the resized volume. 
 
+## Node affinity for an application
 
+To know about node selector refer the [scheduler](/docs/next/scheduler.html) section.
+
+**Modify the application yaml**
+
+We have taken mongo-statefulset application as an example.
+For scheduling application to a particular nodes, we need have labeled nodes, so that mongo application will be schedule on required 
```

Comment:
to a particular nodes = to nodes...
we need have labeled = we must have labeled
will be schedule = will be scheduled